src/DotVVM.Framework/Resources/Scripts/DotVVM.js

```diff
@@ -1870,14 +1897,14 @@ var DotVVM = /** @class */ (function () {
         ko.virtualElements.allowedBindings["dotvvm-SSR-foreach"] = true;
         ko.bindingHandlers["dotvvm-SSR-foreach"] = {
             init: function (element, valueAccessor, _allBindings, _viewModel, bindingContext) {
-                var _a;
                 if (!bindingContext)
                     throw new Error();
                 var value = valueAccessor();
                 var innerBindingContext = bindingContext.extend((_a = {}, _a[foreachCollectionSymbol] = value.data, _a));
                 element.innerBindingContext = innerBindingContext;
                 ko.applyBindingsToDescendants(innerBindingContext, element);
                 return { controlsDescendantBindings: true }; // do not apply binding again
+                var _a;
```

Comment:
For some reason the TypeScript compiler moved the variables around...

I'll try to update my TypeScript compiler since the package is not defined in `package.json`.